src/query/functions-v2/src/scalars/control.rs

```diff
@@ -140,4 +147,78 @@ pub fn register(registry: &mut FunctionRegistry) {
             }),
         }))
     });
+    registry.register_function_factory(IS_NULL, |_, args_type| {
```

Comment:
I suggest using `register_1_arg_core` instead because it supports auto-downcasting. `multi_if` as a variadic function can not use `register_1_arg_core` tho... 

You may implement `is_null` for `NullType` and `NullableType<Generic<>0>` respectively.

Reply:
e.g.

```rust
register.register_1_arg_core::<NullableType<GenericType<0>>, BooleanType, _, _>(
    "is_null",
    FunctionProperty::default(),
    |val| ...,
    |val, _| ...,
);
```

Reply:
thanks, I'm trying to implement it.